src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -299,6 +344,13 @@ private void toggleClimbingSim() {
     Logger.recordOutput("Climber/Climbing", isClimbingStateSim);
   }
 
+  private Command emergencyUndo() {
+    return runOnce(() -> runClimber(ClimberConstants.kClimberEmergencyUndoAngle, 0))
```

Comment:
```suggestion
    return runClimber(ClimberConstants.kClimberEmergencyUndoAngle, 0)
```
runClimber is already a Command